static/js/l10n/i18n.ts

```diff
@@ -0,0 +1,55 @@
+import { createIntl, createIntlCache } from "react-intl";
```

Comment:
/l10n/i18n is pretty awkward.. we should stick with one or the other, maybe stick with i18n?